### ISSUE TITLE
F.E.A.R. (21090)

### ISSUE THREAD

**Issue by d10sfan** [ValveSoftware/Proton#481]
Intro movies work fine, but audio in menu and game does not play. 

System information: https://gist.github.com/d10sfan/cd6abd90b2f5c549f350b9b878c9d300

**Comment by kisak-valve** [comment]
Hello @d10sfan, please copy your system information from steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by d10sfan** [comment]
I've updated the description with the gist.

**Comment by XakepSDK** [comment]
Works fine. Arch Linux.
Sysinfo: https://gist.github.com/XakepSDK/7b776121c7704ff47e116d9c28898a3c

I had trouble with 2 screen setup - after alt-tabbing, while game was in intro state, game window moved to second screen, but mouse was locked in first screen and main menu UI click-boxes were on first screen.
tl;dr - image was on second screen, all interactions with UI were on first screen

**Comment by d10sfan** [comment]
Are you able to hear the audio?

**Comment by XakepSDK** [comment]
Yes, i can. Last 3 switches in settings menu are disabled and forced to off state.
Also, mouse is very very horrible in-game, it feels like reverse acceleration.

**Comment by XakepSDK** [comment]
The issue is that vertical mouse movements are very bad, horizontal is ok. If i move mouse vertically(in-game, not in menu), mouse slow downs.
If i move mouse just horizontally, it's fine, if i move mouse vertically only - vertical movement slow downs.
If i move mouse vertically and horizontally, even horizontal movement slow downs. Feels like dash moving.
Also, i experienced this in other games too - stalker shadow of Chernobyl for example, but this was no so horrifying. I think, these mouse 'bugs' have same source.

**Comment by XakepSDK** [comment]
Game just closed after first level.(ends in helicopter)

**Comment by kisak-valve** [comment]
F.E.A.R. slowdown

Issue transferred from https://github.com/ValveSoftware/Proton/issues/1389.
@dlove67 posted on 2018-09-10T20:56:50:

# Compatibility Report
- Name of the game with compatibility issues: F.E.A.R.
- Steam AppID of the game: 21090

## System Information
- GPU: RX Vega
- Driver/LLVM version: Mesa 18.3.0-Devel LLVM8.0
- Kernel version: 4.18.0
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/dlove67/2420b2af8ff9049f6a9afe4d2db23b53
- Proton version: Proton 3.7-6

## I confirm:
- [X ] that I haven't found an existing compatibility report for this game.
- [X ] that I have checked whether there are updates for my system available.


[steam-21090.log](https://github.com/ValveSoftware/Proton/files/2368329/steam-21090.log)

## Symptoms 
When playing the game and there are particle effects or bodies ragdoll, the FPS will drop to less than 1 fps at times. I found this is particularly bad at a location in the level "Bad Water".

## Reproduction
I have created a save game at and uploaded to google drive here: https://drive.google.com/open?id=1tO5e6CvkyUnQ0i1uowVCRxazDvAoU3PM (the save file is "Bad Water" at time 1:41:54)

You can also see the issue when doing the benchmark, to a much more minor degree, when the soldiers are shooting at each other, framerate drops on my system from around 200+ fps to about 20-30fps.

**Comment by dlove67** [comment]
Small update to my earlier report #1389 with some extra information:

I tried putting all graphics and CPU settings to "Low" in game, but the slowdown was unaffected.

I also tried Vanilla Wine via Lutris (Which had no special fixes, just installed steam and downloaded) and the slowdown was not present, so this appears to be a Proton specific regression.

Also, sorry @kisak-valve. Figured this wouldn't apply since I'm not experiencing the issues reported by @d10sfan. I'll keep it in mind for any future reports I make.

**Comment by shoober420** [comment]
I’m also experiencing mouse stuttering with FEAR. 

The brightness slider also has no effect on the brightness of the game.

https://github.com/shoober420/linux-scripts/blob/main/home/shoober420/sysinfonvidia

Proton 3.7-6 beta

**Comment by shoober420** [comment]
Also, the brightness slider does not seem to effect the brightness in the game. I’m on Arch Linux with the latest proprietary nvidia drivers.

Proton 3.7-6beta

**Comment by necuz** [comment]
[Here's a video of the mouse problem, as I'm experiencing it.](https://www.youtube.com/watch?v=j2hOkcHA4pM) That's what happens when moving the mouse at a constant velocity. [Here's a heavily trimmed log of it running](https://github.com/ValveSoftware/Proton/files/2488483/steam-21090.log), had to remove tons of loading/unloading dinput8.dll to fit within file size limit on uploads here.

- GPU: 1080 Ti
- Driver/LLVM version: 396.54.09
- Kernel version: 4.15.0
- Link to full system information report as Gist: https://gist.github.com/necuz/ed8fdb2dd3a4259ab8be6867847dfc4d
- Proton version: Proton 3.16-2

Edit: I should say that the game works fine on Windows 10, though at like 2-3x the fps so that might be masking the issue if it also exists there.

Edit 2: Using native dinput8.dll feels better, but only when running at obscenely high fps, with VSync the game is unplayable (way worse than what's shown in the video).

**Comment by XakepSDK** [comment]
Mouse stutters only when you move it down.
Running game through XWayland solves this trouble for me.

**Comment by acgaudette** [comment]
I am also experiencing the no audio bug in tandem with the first level stuck glitch. The common solution (extracting WMFadist.exe, copying l3codeca.acm) does not work for me.

**Comment by acgaudette** [comment]
Update: I was able to fix this issue by installing a package with 32-bit mp3 support (specifically [this](https://www.archlinux.org/packages/multilib/x86_64/lib32-mpg123/)).

**Comment by kisak-valve** [comment]
Related: #957

**Comment by dlove67** [comment]
As a note, the libmpg123-0:i386 fix doesn't appear to be the cause of the stuttering issue for me, as I already have that package installed. I could try running this on an Intel PC, as it shows as Gold on https://www.protondb.com/app/21090, and threadripper isn't very well represented there (for instance, there's a known bug where Fallout 4 won't progress past the start screen on it in proton).

**Comment by kisak-valve** [comment]
FEAR

Issue transferred from https://github.com/ValveSoftware/Proton/issues/2486.
@childebert posted on 2019-04-01T20:30:26:

# Whitelist Request
- Name of the game to be whitelisted: FEAR
- Steam AppID of the game: 21090

## System Information

  OS : "Solus" (64 bits)
Nom du noyau : Linux
Version du noyau : 4.20.16-112.current
Version proton : 3.16-8 beta, 4.2-1
 
Pilote : NVIDIA Corporation GeForce GTX 970/PCIe/SSE2
 Version du pilote : 4.6.0 NVIDIA 390.116
 Version OpenGL : 4.6


## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [ ] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [x] I haven't experienced any issues.
- [] There are no issues left open for this game.
- [] Although I consider the gaming experience equal to Windows there are remaining issues:

**Comment by kisak-valve** [comment]
Hello, is anyone seeing an issue with this game and Proton 4.2 that would block it from being whitelisted?

The [steam runtime](https://github.com/ValveSoftware/steam-runtime/commit/d71625fe19d054d61392517b2f9c9febf4b99734) picked up libmpg123 a fair while ago.

**Comment by d10sfan** [comment]
Game works great on this end.

**Comment by dlove67** [comment]
I'm still personally having the slowdown issue, but that may be an issue specific to my hardware (My first guess would be that it doesn't like high thread count CPUs?). I'll post a log when I get home today.

**Comment by dlove67** [comment]
@kisak-valve 

[Here's my proton log](https://github.com/ValveSoftware/Proton/files/3046321/steam-21090.zip)

[My system info:](https://gist.github.com/dlove67/2420b2af8ff9049f6a9afe4d2db23b53)

[and a link to a video I made showing the issue I'm seeing](https://youtu.be/F_gU8CfHKAs?t=70). I'm getting massive framedrops whenever the enemies in the benchmark start shooting, for example.

**Comment by XakepSDK** [comment]
Proton 4.2-2, i still have weird mouse issues. It's feels strange, slowdowns when moving mouse down.

**Comment by shoober420** [comment]
I still have those mouse issues too. I may say that the mouse input code is just poop.

**Comment by dlove67** [comment]
Okay, so it appears setting "Max software sounds" to minimum under performance mostly resolves the issue. The framerate is still lower than I think it should be but it fixes the framerate dropping to 1-2fps. 


Edit: This isn't quite true. It definitely helps a ton, but when multiple enemies are shooting it still drops down to single digit framerates from time to time. I think this is the cause, however.

Here's the log without the framedrops, if it helps.

[steam-21090.zip](https://github.com/ValveSoftware/Proton/files/3051255/steam-21090.zip)

**Comment by shoober420** [comment]
# Compatibility Report
- Name of the game with compatibility issues: F.E.A.R.
- Steam AppID of the game: https://steamdb.info/app/21090/

## System Information
- GPU: GTX 760
- Driver/LLVM version: NVIDIA 440.36
- Kernel version: 5.3.13
- https://github.com/shoober420/linux-scripts/blob/main/home/shoober420/sysinfonvidia
- Proton version: 4.11-8

## Symptoms
Everything is ok, except the mouse input. Its very stuttery and laggy. Literally unplayable to me. There has been mouse input issues with this game since trying this on Wine/Proton. I used "PROTON_USE_D9VK=1" to enable the Vulkan wrapper.

This bug is only apparent for people who use higher polling rates on their mouse. It’s a bug in the engine. More info here:

https://bugs.winehq.org/show_bug.cgi?id=38637

## Reproduction
Play the Game, move around the mouse

**Comment by shoober420** [comment]
# Whitelist Request
- Name of the game to be whitelisted: F.E.A.R.
- Steam AppID of the game: https://steamdb.info/app/21090/

## System Information
- GPU: GTX 760
- Driver/LLVM version: NVIDIA 440.44
- Distro version: Arch
- https://github.com/shoober420/linux-scripts/blob/main/home/shoober420/sysinfonvidia
- Proton version: 4.11-11

Works great out of the box. Mouse stutter bug is now gone. Still highly recommend to use "PROTON_USE_D9VK=1".

**Comment by kisak-valve** [comment]
Hello, is anyone seeing an issue with this game and Proton 4.11-11 that would block it from being whitelisted?

**Comment by SergeyLatyshev** [comment]
FEAR works wine with Proton 4.11 but after upgrading Proton to 5.0 the game started to crash every time when I load a savestate. I switched back to 4.11, it's working now again.

**Comment by kisak-valve** [comment]
Hello @SergeyLatyshev, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by SergeyLatyshev** [comment]
That's strange, but after I switched Proton to 5.0 again, I had no crashes so far. Probably because it was a different level? I had rewritten those save files.
But I got another issue. Weapons look transparent in the player's hands. It didn't happen with 4.11.
![FEAR_20200217232047_1](https://user-images.githubusercontent.com/28780380/74666779-16c73e00-51dd-11ea-9d28-5142af83ce53.jpg)
[steam-21090.zip](https://github.com/ValveSoftware/Proton/files/4214509/steam-21090.zip)

**Comment by kisak-valve** [comment]
Can you check how the weapon renders with `PROTON_USE_WINED3D=1 %command%` in the game's launch options?

**Comment by SergeyLatyshev** [comment]
With this option everything is fine. It seems even the framerate is now higher than before.

**Comment by soredake** [comment]
Game crashes when starting new game. Proton 5.0-2.
[steam-21090.log](https://github.com/ValveSoftware/Proton/files/4233092/steam-21090.log)

**Comment by kisak-valve** [comment]
Hello @soredake, can you check if adding `PROTON_FORCE_LARGE_ADDRESS_AWARE=1 %command%` to the game's launch options helps with your crash?

**Comment by soredake** [comment]
@kisak-valve yes, this helps

**Comment by XakepSDK** [comment]
Does not work with DXVK. Works with WineD3D. Proton 5.0-9
Proton log: https://gist.github.com/XakepSDK/5b5296fe554a2042aa5818b005284c64

**Comment by XakepSDK** [comment]
If you have black screen with DXVK, just wait few mins and/or alt+tab few times. This fixed black screen issue for me. Next start after "fixing" black screen this way was without issue.

**Comment by vide0hanz** [comment]
Revisiting this.

With Proton 7.0 using `PROTON_USE_WINED3D=1` to correct weapon and character model transparency now results in weird looking shadows. Best way I can describe it is that despite running with 32 bit color mode, it appears as if its rendering in 16 bit. 

Without this environment variable, transparent models are still an issue.

**Comment by Blisto91** [comment]
@vide0hanz If you can then also post in [the upstream issue](https://github.com/doitsujin/dxvk/issues/1461) including your system info like GPU and driver version.

**Comment by vide0hanz** [comment]
> @vide0hanz If you can then also post in [the upstream issue](https://github.com/doitsujin/dxvk/issues/1461) including your system info like GPU and driver version.

@Blisto91 done

**Comment by kisak-valve** [comment]
F.E.A.R. When flashlight is shined certain brushes there will be a black texture

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8196.
@UtopicTahu posted on 2024-10-27T23:27:28:

# Compatibility Report
- Name of the game with compatibility issues:F.E.A.R.
- Steam AppID of the game: 21090

## System Information
- GPU: Intel HD 620
- Video driver version:  4.6 Mesa 24.1.7
- Kernel version: 6.11.4-201.fc40.x86_64
- Proton version: 6.3-8 and Later and Experimental

When flashlight is shined at between brushes or at certain brushes there will be a black texture on the brush

## I confirm:
![Screenshot_20241027_190949](https://github.com/user-attachments/assets/557c57d6-22d6-4aed-83f6-c6c931316851)
![Screenshot_20241027_190855](https://github.com/user-attachments/assets/c3f0d8d5-e0b6-4651-9f31-33a98e40b801)

## Symptoms 
[enginemsg.txt](https://github.com/user-attachments/files/17535729/enginemsg.txt)

## Reproduction

**Comment by kisak-valve** [comment]
[21090] F.E.A.R Fails To Start

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8872.
@essays-on-esotericism posted on 2025-06-30T22:33:29:

# Compatibility Report
- Name of the game with compatibility issues: F.E.A.R
- Steam AppID of the game: 21090

## System Information
- GPU: [AMD/ATI] Lucienne (rev c3)
- Video driver version: Mesa 25.1.3
- Kernel version: 6.8.0-57-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/essays-on-esotericism/fb111a985728c74e95b131bb0b7a1983

[steam-21090.zip](https://github.com/user-attachments/files/20987914/steam-21090.zip)

- Proton version: 10 Beta (issue present in all versions after 8)

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



## Symptoms 
F.E.A.R fails to start on Proton versions 9 and 10. It starts and runs fine on Proton 8 however.

## Reproduction
Attempt to run the game

**Comment by kisak-valve** [comment]
Hello @essays-on-esotericism, `err:d3dx:d3dx_pool_sync_shared_parameter Out of memory.` looks like the line of interest from your Proton log. Possibly fixed by https://gitlab.winehq.org/wine/wine/-/commit/1031288361679219d61cfe82c0f4736b36eb7885 upstream.

**Comment by essays-on-esotericism** [comment]
Thanks, it's now working for me on latest Proton 10 release!
